tools/cldr/cldr-to-icu/README.md

```diff
@@ -0,0 +1,140 @@
+<!--
+© 2019 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# Basic instructions for running the LdmlConverter via Maven
+
+
+## Requirements
+
+* A CLDR release for supplying CLDR data and the CLDR API.
+* The Maven build tool
+* The Ant build tool
+
+## Important directories
+
+<dl>
```

Comment:
Nitpick: I would use a table here, to stay 100% markdown, without html.

Reply:
Done. Fair enough, and now that I think about it, tables seem clearer and more concise than definition lists.